x-pack/plugins/maps/public/index_pattern_util.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { indexPatternService } from './kibana_services';
+
+export async function getIndexPatternsFromIds(indexPatternIds) {
```

Comment:
Nice utility function. This can replace logic in [map_embeddable_factory](https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/embeddable/map_embeddable_factory.js#L48)

Reply:
will keep this as-is, since the implementation in `map_embeddable_factory` is a little more sophisticated, swallowing errors on a per-layer basis